### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by tryg-andreas-harmuth** [confluentinc/ksqldb-graphql#18]
Graphql and kafka is something we really want to be using with a library such as this. Is this still actively being maintained/developed?

**Comment by bryantbiggs** [comment]
likewise - looking for NodeJS support and this looks like it would be great but looks like its only a proof of concept so far

**Comment by MrMenezes** [comment]
Nothing has changed so far

**Comment by alexknips** [comment]
I would disagree. This seems pretty abandoned if you cannot build it anymore without resorting to deprecated versions of Apollo server
